pkgs/applications/science/math/scalp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, withGurobi ? false
+, gurobi
+, withCplex ? false
+, cplex
+, withLpsolve ? true
+, lp_solve
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scalp";
+  version = "63";
+
+  src = fetchgit {
+    url = "https://digidev.digi.e-technik.uni-kassel.de/git/scalp.git";
```

Comment:
What git server are they using if you can't browse it on the web? Does not give much trust.

Reply:
Well, at least FloPoCo trusted the link in their [install script](http://flopoco.org/install_scripts/install-full-gitmaster-on-focal64.sh)...

Reply:
Never heard of that either which does not mean much

Reply:
I just created a [mirror](https://github.com/wegank/scalp) on GitHub. Should the package use it instead?

Reply:
That git repo is hosted in a German university, and the software has been part of a few publications, I assume it can be trusted.

Reply:
Actually, using a third-party mirror now seems less trusty, since it's not published by the authors.

Reply:
I've put the mirror in the comment, so we always have a fallback with the same hash.

Reply:
@SuperSandro2000 this needs your approval in order to move forward.

Reply:
No, it does not necessarily require my review.

Reply:
Not as sketchy as the currently-down `gitlab.freedesktop.org` at least...

Reply:
gitlab.freedesktop.org, the gitlab instance of freedesktop which has half of x11, wayland and more is not sketchy at all.

Reply:
I mean, their instance should have been less sketchy than a random link from a random university, but it was literally down on [June 12](https://people.freedesktop.org/~cbrill/dri-log/index.php?channel=freedesktop&highlight_names=&date=2022-06-12) and [July 30](https://people.freedesktop.org/~cbrill/dri-log/index.php?channel=freedesktop&highlight_names=&date=2022-07-30), each time for over 10 hours.

Reply:
sketchy means that the posted link above has no information at all except that git repo and happens to be a 4th level subdomain of an university. That could almost be anything. Where is the project page, contact information, web git, something?

Reply:
The project page is [here](https://digidev.digi.e-technik.uni-kassel.de/scalp/) (though also 4th level subdomain...)